src/components/arpfingerprint/arpfingerprint.tsx

```diff
@@ -0,0 +1,156 @@
+import { Button, Stack, TextInput, Alert } from "@mantine/core";
+import { useForm } from "@mantine/form";
+import { useCallback, useState } from "react";
```

Comment:
You need to import useEffect so you can check if the command is available. 

- [ ] Import useEffect.